### ISSUE TITLE
Post Processing in v21

### ISSUE THREAD

**Issue by Trusiak** [enable3d/enable3d#55]
Hello,

Due to the introduction of the double canvas element, is it now possible to implement post-process solutions to enable3d?

**Comment by yandeu** [comment]
I will look into it before releasing v21.

**Comment by yandeu** [comment]
I just added support for post-processing. It will be available in version 0.21.0.

Once `this.composer` is defined, the update loop will do `composer.render()` instead of `renderer.render()`:
```js
if (this.composer) this.composer.render()
else this.renderer.render(this.scene, this.camera)
```

You will be able to do this:
```js
import { EffectComposer, RenderPass, GlitchPass, DotScreenShader, ShaderPass } from 'enable3d'

create() {
  this.warpSpeed()

  this.composer = new EffectComposer(this.renderer)

  const renderPass = new RenderPass(this.scene, this.camera)
  this.composer.addPass(renderPass)

  const effect = new ShaderPass(DotScreenShader)
  effect.uniforms['scale'].value = 4
  this.composer.addPass(effect)

  const glitchPass = new GlitchPass()
  this.composer.addPass(glitchPass)
}
```
